README.md

```diff
@@ -307,35 +323,16 @@ There are two ways of running Reseller Component Sample:
 
 ### FIDO IoT Reseller REST APIs
 
-1. GET /api/v1/resell/vouchers/<serial_number>
-
-  Returns the extended voucher for the given serial number from `RT_DEVICES` table.
-
-2. POST /api/v1/resell/vouchers/<serial_number>
-
-  Adds voucher to `RT_DEVICES` table against the serial number.
-
-3. DELETE /api/v1/resell/vouchers/<serial_number>
-
-  Deletes voucher from `RT_DEVICES` table with the specified serial number.
-
-4. POST /api/v1/resell/customers/<customer_id>
-
-  Adds customer keyset to `RT_CUSTOMERS` table.
-
-5. POST /api/v1/resell/customers/<customer_id>
+| Operation                      | Description                        | Path/Query Parameters    | Content Type   |Request Body  | Response Body |
+| ------------------------------:|:----------------------------------:|:------------------------:|:--------------:|-------------:|--------------:|
+| GET /api/v1/resell/vouchers/<serial_number>?id=<customer_id> | Assigns the customer and returns the extended voucher for the given serial number from `RT_DEVICES` table. | Query - id: Customer Id, Path - Device Serial Number | | | Ownership voucher |
+| POST /api/v1/resell/vouchers/<serial_number> | Adds voucher to `RT_DEVICES` table against the serial number. | Path - Device Serial Number | application/cbor | Ownership voucher | |
+| DELETE /api/v1/resell/vouchers/<serial_number> | Deletes voucher from `RT_DEVICES` table with the specified serial number. | Path - Device Serial Number | | | |
+| POST /api/v1/resell/customers/?id=<customer_id>&name=<customer_name> | Adds customer keyset to `RT_CUSTOMERS` table. | Query - id: Customer Id, name: Customer Name | text/plain; charset=us-ascii | Customer PEM formatted Public keys | |
+| POST /api/v1/resell/keys/?alias=<alias> | Adds new Reseller keys to the keystore with the given alias | Query - alias: Alias to be used in keystore | | PEM formatted certificate and private key | | 
+| DELETE /api/v1/resell/keys/?alias=<alias> | Deletes the keys corresponding to the input alias from keystore. | Query - alias: Alias to be used in keystore | | | |
```

Comment:
knitpick: Query - alias: Alias to be used in keystore -> Query - alias: Alias to be removed from the keystore